joomla/joomla-cms
# [4.3] Media Manager Ordering

Pull Request for Issue # .

### Summary of Changes
- Base work for the ordering
- Supports: name, size, dimensions, date created and date modified

~The TODO here is to expose to the icons view the ui (ie dropdown, series of buttons, etc) to control the ordering. Right now you have to switch to the table view to select another column for ordering...~ 


### Testing Instructions
Apply the PR and run `npm ci`

Check the grid layout of the media browser and click on the table headers


### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request
![Screenshot 2023-01-29 at 13 29 48](https://user-images.githubusercontent.com/3889375/215326219-c49f9a6d-f0c2-4290-9ca3-878124200516.png)
![Screenshot 2023-01-29 at 13 30 11](https://user-images.githubusercontent.com/3889375/215326233-c4333a27-574b-4bd9-a9ec-8b9a48126809.png)



### Link to documentations
Please select:
